src/transaction/log_page_buffer.c

```diff
@@ -7359,26 +7362,26 @@ logpb_checkpoint_trantable (THREAD_ENTRY * const thread_p)
 	_er_log_debug (ARG_FILE_LINE, "checkpoint_trantable: failed; writing metalog to file\n");
 	return res_metalog_to_file;
       }
+  }
 
-    logpb_flush_pages (thread_p, &trantable_checkpoint_lsa);
+  logpb_flush_pages (thread_p, &trantable_checkpoint_lsa);
```

Comment:
I explained in the PR text about this change.
Is it safe?
If not, why?
If it is, why?